SECURITY.md

```diff
@@ -121,15 +121,15 @@ any breach.
 ## Vulnerabilities in TensorFlow
 
 TensorFlow is a large and complex system. It also depends on a large set of
-third party libraries (e.g., `numpy`, `libjpeg-turbo`, PNG parsers, `protobuf`).
-It is possible that TensorFlow or its dependent libraries contain
+third-party libraries (e.g., `numpy`, `libjpeg-turbo`, PNG parsers, `protobuf`).
+TensorFlow or its dependent libraries may contain
```

Comment:
Not a relevant change. Please revert.

Also, here you have not replaced "dependent" as you did above.